exporter/src/main.rs

```diff
@@ -57,28 +62,30 @@ fn take_screenshot(
     )?;
 
     let mut result = Vec::new();
-    for i in 0..frames {
+    let totalframes = frames + skipframes + 1;
+
+    for i in 0..totalframes {
         if let Some(progress) = &progress {
             progress.set_message(&format!(
                 "{} frame {}",
                 swf_path.file_stem().unwrap().to_string_lossy(),
                 i
             ));
         }
-
         player.lock().unwrap().run_frame();
-        player.lock().unwrap().render();
-
-        let mut player = player.lock().unwrap();
-        let renderer = player
-            .renderer_mut()
-            .downcast_mut::<WgpuRenderBackend<TextureTarget>>()
-            .unwrap();
-        let target = renderer.target();
-        if let Some(image) = target.capture(renderer.device()) {
-            result.push(image);
-        } else {
-            return Err(format!("Unable to capture frame {} of {:?}", i, swf_path).into());
+        if skipframes == 0 || i > skipframes {
```

Comment:
Could it just be if `i >= skipframes`?

Reply:
Doh! I prefer your suggestion much better. :) 


I haven't used Github's suggestion to directly commit code before so this is an attempt at this.
```suggestion
        i >= skipframes {
```